packages/calcite-components/src/components/action/action.tsx

```diff
@@ -105,6 +105,13 @@ export class Action extends LitElement implements InteractiveComponent {
   /** Specifies the size of the component. */
   @property({ reflect: true }) scale: Scale = "m";
 
+  /**
+   * When `full`, the component's width spans all its parent's available space
+   *
+   * @private
+   */
+  @property({ reflect: true }) width: "auto" | "full" = "auto";
```

Comment:
Can you import and use the `Width` type from `interfaces` and extract only "auto" and "full"?

src/components/interfaces.ts